website/docs/r/virtual_machine.html.markdown.orig

```diff
@@ -0,0 +1,1637 @@
+---
```

Comment:
Did you mean to include a file that ends with `.orig`?

Reply:
I didn't. Thanks for catching that! Got it removed now.